nvblox_ros/src/nvblox_node_main.cpp

```diff
@@ -41,6 +41,7 @@ int main(int argc, char * argv[])
   //std::shared_ptr<nvblox::NvbloxNode> node(new nvblox::NvbloxNode());
   nvblox::NvbloxNode NvbloxNode(nh);
 
+  // TODO(TT) Make sure to check the number of subscribers to the topics.
   ros::MultiThreadedSpinner spinner(4);
```

Comment:
I don't think we need 4 threads, since only 1 thread can really touch the GPU at a time. So all the subscribers can be on 1 thread, and main processing on a second. I can change this next PR.